src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -130,6 +133,57 @@ public void should_attach_exception_record_to_the_existing_case_with_evidence_do
         verifyExistingCaseIsUpdatedWithExceptionRecordData(caseDetails, exceptionRecord, 1);
     }
 
+    @Test
+    public void should_attach_supplementary_evidence_with_pending_payments() throws Exception {
+        //given
+        CaseDetails caseDetails = ccdCaseCreator.createCase(emptyList(), now()); // with no scanned documents
+        CaseDetails exceptionRecord = createExceptionRecord(
+            "envelopes/supplementary-evidence-envelope-with-payment.json"
+        );
+
+        // when
+        attachExceptionRecord(caseDetails, exceptionRecord, null);
+
+        //then
+        await("Exception record with pending payments is attached to the case")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(() -> isExceptionRecordAttachedToTheCase(caseDetails, 1));
+
+        verifyExistingCaseIsUpdatedWithExceptionRecordData(caseDetails, exceptionRecord, 1);
+    }
+
+    @Disabled("Functionality not implemented yet")
+    @Test
+    public void should_return_error_when_attaching_exception_with_pending_payments() throws Exception {
+        //given
+        CaseDetails caseDetails =
+            ccdCaseCreator.createCase(
+                singletonList(
+                    new Document(
+                        "certificate1.pdf", "154565768", "other", null, Instant.now(), documentUuid, Instant.now()
+                    )),
+                Instant.now()
+            ); // with 1 scanned document
+
+        CaseDetails exceptionRecord = createExceptionRecord(
+            "envelopes/exception-classification-envelope-with-payment.json"
+        );
+
+        // when
+        Response response = invokeCallbackEndpoint(caseDetails, exceptionRecord, null, true);
+
+        // then
+        List<String> errors = response.jsonPath().getList("errors");
+        assertThat(errors).isNotEmpty()
+            .contains("The 'attach to case' event is not supported for the Exception Record with pending payments");
```

Comment:
Has the message already been agreed? I was thinking that we could say something like `Cannot attach this exception record to a case because it has pending payments`. Lack of support might lead the caseworker into thinking that something wasn't implemented, yet.

Reply:
This sounds better. I'll update